### PR TITLE
Add 2nd HW I2C support

### PR DESCRIPTION
**Pull request by benoitm974**
As I needed to use the second I2C channel on ESP32 I propose a change to SSD1306Wire file.

### CODE REVIEW

**Comment by marcelstoer** [comment]
Please add a note about this feature to the README e.g. at https://github.com/ThingPulse/esp8266-oled-ssd1306#i2c-with-wireh.

**Comment by benoitm974** [comment]
Upated README.md

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ The library supports different protocols to access the OLED display. Currently t
 SSD1306Wire display(0x3c, SDA, SCL);  // ADDRESS, SDA, SCL
 // for 128x32 displays:
 // SSD1306Wire display(0x3c, SDA, SCL, GEOMETRY_128_32);  // ADDRESS, SDA, SCL, GEOMETRY_128_32 (or 128_64)
+// for using 2nd Hardware I2C (if availbale)
```
Comment (thijstriemstra) [inline README.md:79]: typo here, should say 'available'

Comment (benoitm974) [reply README.md]: Done.

**Comment by thijstriemstra** [comment]
@benoitm974 can you also add this for SH1106?

**Comment by benoitm974** [comment]
> @benoitm974 can you also add this for SH1106?

ok it's done for SH1106

**Comment by thijstriemstra** [comment]
awesome, thanks, works great.

**Comment by marcelstoer** [comment]
Benoit, thanks for writing such clear in-code documentation.

File: README.md [hunk 2]
```diff
@@ -84,6 +89,9 @@ for a SH1106:
 #include "SH1106Wire.h"
 
 SH1106Wire display(0x3c, SDA, SCL);  // ADDRESS, SDA, SCL
+// By default SD1306Wire set I2C frequency to 700000, you can use set either another frequency or skip setting the frequency by providing -1 value
```
Comment (thijstriemstra) [inline README.md:92]: this should be SH1106Wire. I would use 'By default the I2C frequency of SH1106Wire is ' etc

Comment (benoitm974) [reply README.md]: Done.

**Pull request merged**